tests/nfv/vsperf_installation.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: openvswitch installation and CLI test
+#
+#   This test does the following
+#    - Installs openvswitch (dpdk if DPDK==1)
+#    - Starts the systemd service unit
+#    - Executes a few basic openvswitch commands
+#
+# Maintainer: Jose Lausuch <jalausuch@suse.de>
+
+use base "consoletest";
+use testapi;
+use strict;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    zypper_call('in git', timeout => 200);
+
+    # Clone repository
+    assert_script_run "git clone https://gerrit.opnfv.org/gerrit/vswitchperf";
```

Comment:
Just for flexibility can you extract this to a variable ?

Reply:
done